metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -302,71 +303,112 @@ public void remove(final int index) {
 
         private void removeNested(final String[] fields) {
             if (fields.length >= 1 && fields[0].equals(ASTERISK)) {
-                for (int i = 0; i < size(); ++i) {
-                    remove(i);
-                }
+                list.clear();
             }
             else if (fields.length >= 1 && isNumber(fields[0])) {
-                final int index = Integer.parseInt(fields[0]) - 1;
+                final int index = Integer.parseInt(fields[0]) - 1; // TODO: 0-based Catmandu vs. 1-based Metafacture
```

Comment:
I can't think of anything off the top of my head. Where is Metafacture 1-based? Seeing as it doesn't have a concept of arrays...

Reply:
I was probably thinking of the way JsonDecoder numbers records and array elements.

Reply:
Ah, okay, makes sense. Not sure, though, if both domains (names and indexes) _have_ to be congruent at all.